projectDocs/translating/crowdin.md

```diff
@@ -64,9 +64,27 @@ Some comments provide an example output message to help you understand what NVDA
 
 ## Translating NVDA's interface
 
-* Download nvda.po from the Files section of your language on Crowdin.
-* Open the po file in Poedit, translate, and save the file.
-* Upload the po file back to Crowdin.
+* Download the po file for your language from  Crowdin using NVA's l10nUtil.exe:
+```
+l10nUtil.exe downloadTranslationFile <language> <crowdinFilePath> [<localFilePath>]
+```
+E.g.
+```
+l10nUtil.exe downloadTranslationFile fr nvda.po
+```
+The first time you will be asked for an authorization token.
+Please visit https://crowdin.com/settings#api-key
+And create a personal access token, Ensuring that it has at least the translations scope.  Then paste this into the user prompt.
+this will be saved in ~/.nvda_crowdin for future use.
```

Comment:
```suggestion
Please visit [Crowdin's API page](https://crowdin.com/settings#api-key) and create a Personal Access Token.
Ensure that it has at least the translations scope.
Then paste this into the user prompt.
This will be saved in ~/.nvda_crowdin for future use.
```